src/app/pages/server-details/server-details.component.scss

```diff
@@ -58,6 +58,22 @@
   }
 }
 
+#toast-danger {
+  position: fixed;
+  top: 3rem;
+  right: 1rem;
+  z-index: 200;
+  transform: translateX(100%);
+  transition: transform 0.5s ease-out;
+  display: none;
+  /* other styles... */
+}
+
+#toast-danger.show {
+  transform: translateX(0);
+  display: flex;
+}
+
```

Comment:
IMO we need to add CSS for the `li` tags as well to make it visible that it's a list:

![image](https://github.com/user-attachments/assets/b2b8ed05-6df4-47d0-8b2a-7fd8da82f754)